remoteappmanager/templates/admin_page.html

```diff
@@ -1,6 +1,6 @@
 {% extends "page.html" %}
 
-{% set skin = "skin-red" %}
```

Comment:
With tornado `{% set skin = ... %}` only sets a local variable which is not available in the parent scope. Unfortunately, according to https://github.com/tornadoweb/tornado/pull/1323, an equivalent implementation in Tornado of the behavior of Jinja is not available yet. A workaround is to use a block.